org.eclipse.jdt.ui/core extension/org/eclipse/jdt/internal/corext/callhierarchy/CallHierarchy.java

```diff
@@ -60,25 +56,25 @@ public static CallHierarchy getDefault() {
     public boolean isSearchUsingImplementorsEnabled() {
         IPreferenceStore settings = JavaPlugin.getDefault().getPreferenceStore();
 
-        return settings.getBoolean(PREF_USE_IMPLEMENTORS);
+        return settings.getBoolean(CallHierarchyCore.PREF_USE_IMPLEMENTORS);
```

Comment:
static import CallHierarchyCore.* instead?

Reply:
Better all 4 static imports since `org.eclipse.jdt.ui.staticondemandthreshold=99` (in `org.eclipse.jdt.core.manipulation/.settings/org.eclipse.jdt.ui.prefs`)

```java
import static org.eclipse.jdt.internal.corext.callhierarchy.CallHierarchyCore.PREF_FILTERS_LIST;
import static org.eclipse.jdt.internal.corext.callhierarchy.CallHierarchyCore.PREF_FILTER_TESTCODE;
import static org.eclipse.jdt.internal.corext.callhierarchy.CallHierarchyCore.PREF_USE_FILTERS;
import static org.eclipse.jdt.internal.corext.callhierarchy.CallHierarchyCore.PREF_USE_IMPLEMENTORS;
```

Reply:
i have imported all 6 constants to callHierarchy.